py/bitbox02/bitbox02/communication/bitbox_api_protocol.py

```diff
@@ -704,9 +706,12 @@ def get_info(transport: TransportLayer) -> Tuple[str, Platform, Union[BitBox02Ed
         else:
             raise Exception("Unknown platform: {}".format(platform))
 
-        unlocked_byte = response[0]
+        unlocked_byte, response = response[0], response[1:]
         unlocked = {0x00: False, 0x01: True}[unlocked_byte]
-        return (version_str, platform, edition, unlocked)
+
+        initialized_byte = response[0]
+        initialized = {0x00: False, 0x01: True}[initialized_byte]
```

Comment:
This will crash on firmware versions that don't support this yet. Maybe one way of dealing with this is to return `Optional[bool]` and document that it is `None` if not supported.

Reply:
ah yes thanks, it would throw index out of range for `initialized_byte = response[0]` should I just `try except` it and set `initialized` to `None` in `except` or is there a better way of handling this (checking somehow if it is supported ?)

Reply:
You can check the firmware version, there are other examples. For now let's expect this to go into v9.20.0, but we can adjust the number if it lands quicker.